projects/packages/jitm/src/js/jetpack-jitm.js

```diff
@@ -306,14 +302,19 @@ jQuery( document ).ready( function ( $ ) {
 
 			var full_jp_logo_exists = $( '.jetpack-logo__masthead' ).length ? true : false;
 
-			$.get( window.jitm_config.api_root + 'jetpack/v4/jitm', {
-				message_path: message_path,
-				query: query,
-				full_jp_logo_exists: full_jp_logo_exists,
-				_wpnonce: $el.data( 'nonce' ),
+			// Site ID will be automatically added to the request.
+			const url = `/wpcom/v2/jitm-v2`;
```

Comment:
Can we move this to the top and reuse in https://github.com/Automattic/jetpack/pull/41252/files#diff-1800a11dbdc47e291367545ea09cb52c06f5a5aebcb570202b557004393b979bR160 ?

Reply:
Yes 👍